modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_drop-down-filter.scss

```diff
@@ -0,0 +1,4 @@
+.drop-down-menu-items {
+	padding: 0 !important;
```

Comment:
use `p-0` in classNames instead of using here

Reply:
done!